### ISSUE TITLE
feat(analytics): add logTransaction method

### ISSUE THREAD

**Issue by TomDoesb** [capawesome-team/capacitor-firebase#629]
### Plugin(s)

- [X] Analytics
- [ ] App
- [ ] App Check
- [ ] Authentication
- [ ] Crashlytics
- [ ] Cloud Firestore
- [ ] Cloud Messaging
- [ ] Cloud Storage
- [ ] Performance
- [ ] Remote Config

### Version

6.0.0

### Platform(s)

- [ ] Android
- [X] iOS
- [ ] Web

### Current behavior

Firebase documentation states the following on tracking in app purchases with storekit 2
<img width="1044" alt="image" src="https://github.com/capawesome-team/capacitor-firebase/assets/12149839/88574190-22f5-4af0-9614-f5844281f78a">

However the `logTransaction()` is not available in `capacitor-firebase` and logging the event using 
`FirebaseAnalytics.logEvent({name: 'in_app_purchase', params: {productIdentifier: productIdentifier}});`

gives the following log message: `10.24.0 - [FirebaseAnalytics][I-ACS013006] Event name is reserved. Ignoring event: in_app_purchase`

### Expected behavior

I would expect to be able to log an iOS StoreKit 2 purchase. This is now not possible because google analytics expects the key_event name to be `in_app_purchase` which is reserved..

### Reproduction

-

### Steps to reproduce

-

### Other information

Docs: https://firebase.google.com/docs/analytics/measure-in-app-purchases

### Capacitor doctor

💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 6.0.0
  @capacitor/core: 6.0.0
  @capacitor/android: 6.0.0
  @capacitor/ios: 6.0.0

Installed Dependencies:

  @capacitor/android: not installed
  @capacitor/cli: 6.0.0
  @capacitor/ios: 6.0.0
  @capacitor/core: 6.0.0

[success] iOS looking great! 👌

### Before submitting

- [X] I have read and followed the [bug report guidelines](https://capawesome.io/contributing/bug-reports/).
- [X] I have attached links to possibly related issues and discussions.
- [X] I understand that incomplete issues (e.g. without reproduction) are closed.

**Comment by EinfachHans** [comment]
@robingenz with storekit 1 officially being deprecated it would be awesome to somehow support the logging of storekit 2 purchases

**Comment by robingenz** [comment]
Yes, I'll have to have a look at it, but I have a few high prio todos at the moment. :/

**Comment by leng-yue** [comment]
Any update?

**Comment by robingenz** [comment]
Not yet but PRs are welcome.
